src/index.js

```diff
@@ -88,18 +92,41 @@ function visitEntity(entity, entitySchema, bag, options) {
   let normalized = visitObject(entity, entitySchema, bag, options);
   mergeIntoEntity(stored, normalized, entityKey);
 
+  if(entitySchema._mappedBy) {
+    const fk = entitySchema._mappedBy;
+    if(entitySchema[fk] instanceof IterableSchema && parentId) {
+      if(!stored[fk]) {
+        stored[fk] = [];
+      }
+      stored[fk] = Array.from(new Set([
```

Comment:
It’s good to refer to MDN for such cases, e.g. https://developer.mozilla.org/en/docs/Web/JavaScript/Reference/Global_Objects/Set

<img width="463" alt="screen shot 2016-04-07 at 16 13 17" src="https://cloud.githubusercontent.com/assets/810438/14356165/ab81f220-fcdb-11e5-9530-4442aeea43dc.png">